assets/js/base/components/product-price/stories/index.tsx

```diff
@@ -23,6 +23,7 @@ export default {
 	},
 	args: {
 		align: 'left',
+		currency: currencies.EUR,
```

Comment:
For some reason I had missed this. Thanks @thealexandrelara ! I have implemented your proposed change, could you please review it once more?

Reply:
Sure, thank you @sunyatasattva  for solving this, I just tested and now the selector is working 🎉 . The only minor issue that I saw is that when the page is first loaded, no option is selected by default:

<img width="2656" alt="image" src="https://user-images.githubusercontent.com/20469356/212986834-d28b59c1-d325-403d-937b-9626d12e7020.png">

It seems that the expected value for the `currency` property is now the `currencies` object key (i.e.: EUR, USD):

```
args: {
		align: 'left',
		currency: 'EUR'
        ...
};

```
Doing that will trigger TypeScript warnings on the file `assets/js/base/components/product-price/index.tsx` (line 194), because the currency property is expecting one of these types instead of a string:
```
currency?: Currency | Record< CurrencyCode, never >;
```

In this case, I think we have two options:
1. Update line 194 on the file above; however, in this case we would be editing the "production code" just to make it work with Storybook;
2. The second option is to extend `ProductPriceStoryProps` on file `assets/js/base/components/product-price/stories/index.tsx` and override the `currency` property with the `CurrencyCode` type, something like this:
```
import { CurrencyCode } from '@woocommerce/types';

[...]

interface ProductPriceStoryProps extends Omit< ProductPriceProps, 'currency' > {
	currency: CurrencyCode;
}

export default {
	title: 'WooCommerce Blocks/@base-components/ProductPrice',
	component: ProductPrice,
	argTypes: {
		align: {
			control: { type: 'radio' },
			options: ALLOWED_ALIGN_VALUES,
		},
		currency: currencyControl,
	},
	args: {
		align: 'left',
		currency: 'EUR',
		format: '<price/>',
		price: 3000,
	},
} as Meta< ProductPriceStoryProps >;
```

These are just suggestions, maybe you know a better way to solve this